src/aks-preview/azext_aks_preview/tests/latest/test_managed_cluster_decorator.py

```diff
@@ -1189,477 +1189,6 @@ def test_get_image_cleaner_interval_hours(self):
         with self.assertRaises(MutuallyExclusiveArgumentError):
             ctx_5.get_image_cleaner_interval_hours()
 
-    def test_get_enable_azure_keyvault_kms(self):
```

Comment:
Do we need to remove the tests in src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py? 
I guess no. Because we need them to ensure aks-preview without kms codes can work well with azure-cli core. Right?

Reply:
Good point. Tests in aks-preview do help us ensure aks-preview without kms codes can work well with azure-cli core. However, sometimes some tests use additional patches, which need to be slightly modified to execute normally. For KMS related test cases, they could run correctly.